app/components/domain-form.tsx

```diff
@@ -0,0 +1,51 @@
+import { AddIcon, InfoIcon } from '@chakra-ui/icons';
+import { Button, HStack, Input, Select, Textarea, Tooltip, VStack } from '@chakra-ui/react';
+import { Form } from '@remix-run/react';
+
+export default function DomainForm() {
+  return (
+    <Form className="domain-form" method="post">
+      <VStack maxW="xl" spacing="5">
+        <HStack w="100%">
+          <Input placeholder="Domain Name" name="name" />
+          <Tooltip>
+            <InfoIcon></InfoIcon>
+          </Tooltip>
+        </HStack>
+        <HStack w="100%">
+          <Select placeholder="Type" name="type">
+            <option value="A">A</option>
+            <option value="AAAA">AAAA</option>
+            <option value="CNAME">CNAME</option>
+            <option value="TXT">TXT</option>
+          </Select>
+          <Tooltip>
+            <InfoIcon></InfoIcon>
+          </Tooltip>
+        </HStack>
+        <HStack w="100%">
+          <Input placeholder="Value" name="value" />
+          <Tooltip>
+            <InfoIcon></InfoIcon>
+          </Tooltip>
+        </HStack>
+        <HStack w="100%">
+          <Input placeholder="Ports" name="ports" />
+          <Tooltip>
+            <InfoIcon></InfoIcon>
+          </Tooltip>
+        </HStack>
+        <HStack w="100%">
+          <Input placeholder="Course" name="course" />
+          <Tooltip>
+            <InfoIcon></InfoIcon>
+          </Tooltip>
+        </HStack>
+        <Textarea placeholder="Description" rows={10} name="description" />
+      </VStack>
+      <Button type="submit" mt="6" rightIcon={<AddIcon boxSize={3.5} mt="0.15rem" />}>
```

Comment:
We probably should avoid hardcoding our spacing, and use the default that chakra provides

Reply:
I did not see any chakra spacing that was small enough for this, so I went with a hardcoded value

Reply:
https://chakra-ui.com/docs/styled-system/theme#spacing

Check it out it has `0.5` spacing

Reply:
With `0.5` spacing, that 0.025 rem difference makes the plus sign slightly off center